tensorflow/lite/micro/riscv32_baremetal/README.md

```diff
@@ -0,0 +1,38 @@
+### **RISC-V 32-Bit Bare-Metal Support**
```

Comment:
The current setup appears to be highly specific to QEMU. If this is the case, perhaps we should rename `riscv32_baremetal` to `riscv32_qemu` for better clarity.